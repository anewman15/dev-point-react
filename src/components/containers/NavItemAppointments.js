import { Link } from 'react-router-dom';

function NavItemAppointments() {
  return (
    <div className="navbar-item">
      <Link className="text-white" to="/appointments">Your Appointments</Link>
    </div>
  );
}

export default NavItemAppointments;
