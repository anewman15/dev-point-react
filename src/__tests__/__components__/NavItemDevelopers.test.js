import { unmountComponentAtNode } from 'react-dom';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavItemAppointments from '../../components/containers/NavItemAppointments';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('NavItemDevelopers', () => {
  it('renders the NavItemDevelopers component to the dom', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <NavItemAppointments />
        </MemoryRouter>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
