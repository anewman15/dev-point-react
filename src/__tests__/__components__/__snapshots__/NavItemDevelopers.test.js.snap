// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItemDevelopers renders the NavItemDevelopers component to the dom 1`] = `
<div
  className="navbar-item"
>
  <a
    className="button is-warning has-text-white has-text-weight-bold is-fullwidth"
    href="/appointments"
    onClick={[Function]}
  >
    Your Appointments
  </a>
</div>
`;
